src/Microsoft.Azure.WebJobs.Host/Bindings/AttributeCloner.cs

```diff
@@ -311,6 +305,44 @@ internal TAttribute ResolveFromBindings(IReadOnlyDictionary<string, object> bind
             return newAttr;
         }
 
+        // Resolution policy for { } in  binding templates. 
+        // The default policy is just a direct substitution for the binding data. 
+        // Derived policies can enforce formatting / escaping when they do injection. 
+        private class ResolutionPolicy
+        {
+            public virtual string TemplateBind(BindingTemplate template, IReadOnlyDictionary<string, object> bindingData)
+            {
+                if (bindingData == null)
+                {
+                    return template.Pattern;
+                }
+                return template.Bind(bindingData);
+            }
+
+            // Expand on [AutoResolve] property to make this pluggable, 
+            public static ResolutionPolicy GetPolicy(PropertyInfo propInfo)
+            {
+                if (propInfo.DeclaringType == typeof(TableAttribute) && propInfo.Name == "Filter")
```

Comment:
Yes. This is a a highly tactical change to get Table.Filter in quickly.  Note that we could revert a bunch of files around bindings, bindng provider, flow the delegates, etc - those changes are more complex and not necessary to fix table.filter. 

---
In reply to: [98059901](https://github.com/Azure/azure-webjobs-sdk/pull/984#discussion_r98059901) [](ancestors = 98059901)